client/src/App.vue

```diff
@@ -36,7 +36,11 @@ export default defineComponent({
   async mounted() {
     // Fetch data needed to render data layers and update state.
     if (this.state != null) {
-      const dataLayers = [leadServiceLinesByWaterSystemLayer, leadAndCopperViolationsByCountyDataLayer, populationByCountyDataLayer];
+      const dataLayers = [
```

Comment:
Idk which PR will submit first but I think you can remove this.fetchInitialData and then set the state in setup. See: https://github.com/BlueConduit/open-data-platform/pull/49/files#diff-ecdb84b8617d5229ab0f8338eab22ddbfcfd993c52796b7d05605245e82428f4

It requires setting the data layer in mounted() in MapView